components/docker-auth/pkg/extension/acl.go

```diff
@@ -20,239 +20,229 @@ package extension
 
 import (
 	"database/sql"
-	"encoding/json"
 	"errors"
 	"fmt"
-	"log"
-	"net"
 	"strings"
 
+	"go.uber.org/zap"
+
+	"github.com/cesanta/docker_auth/auth_server/api"
 	_ "github.com/go-sql-driver/mysql"
 )
 
-type Labelstest map[string][]string
+func ValidateAccess(db *sql.DB, actions []string, username string, repository string, labels api.Labels,
```

Comment:
Shall we change this to isUserAuthorized ? Always try to use meaningful short names for methods

Reply:
Fixed by #159